src/App.tsx

```diff
@@ -30,31 +123,7 @@ export const App = () => {
       </form>
 
       <section className="TodoList">
```

Comment:
This section tag should be placed in the TodoList file, so you will get rid of empty tags in that file